### PR TITLE
Support to fix bug#1094157 and bug#1094963

### PR DESCRIPTION
**Pull request by ancorgs**
Both bugs are about adapting the DeviceMapper names of the encrypted devices. These changes will allow us to proceed on https://github.com/yast/yast-storage-ng/pull/660 and https://github.com/yast/yast-storage-ng/pull/659

### CODE REVIEW

File: storage/Devices/EncryptionImpl.h [hunk 1]
```diff
@@ -56,6 +56,13 @@ namespace storage
 
 	virtual string get_displayname() const override { return get_dm_table_name(); }
 
+	/**
+	 * Redefined from BlkDeviceImpl to ensure the device name is in sync
+	 * with the DeviceMapper name, since the kernel device names for
+	 * Encryption devices are inferred from their DeviceMapper names.
+	 */
+	void set_dm_table_name(const string& dm_table_name);
```
Comment (joseivanlopez) [inline storage/Devices/EncryptionImpl.h]: NP: I usually add `virtual` in the sub-classes too, to make it more explicit. If you only look at `EncryptionImpl` you don't know if `set_dm_table_name` is virtual or not. But only a matter of taste.

Comment (ancorgs) [reply storage/Devices/EncryptionImpl.h]: Fixed. I'm a C++ noob.

File: storage/Devices/EncryptionImpl.h [hunk 2]
```diff
@@ -56,6 +56,13 @@ namespace storage
 
 	virtual string get_displayname() const override { return get_dm_table_name(); }
 
+	/**
+	 * Redefined from BlkDeviceImpl to ensure the device name is in sync
+	 * with the DeviceMapper name, since the kernel device names for
+	 * Encryption devices are inferred from their DeviceMapper names.
+	 */
+	virtual void set_dm_table_name(const string& dm_table_name);
```
Comment (aschnell) [inline storage/Devices/EncryptionImpl.h]: Please add ```override```.

**Comment by aschnell** [comment]
LGTM

**Pull request merged**